help/destinations/catalog/email-marketing/mailchimp-interest-categories.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Mailchimp Interest Categories
+description: The Mailchimp Interest Categories destination allows you to export your contacts data and activate it within Mailchimp. The contacts can be managed as segments based on their interest category subscriptions.
+---
+# [!DNL Mailchimp Interest Categories] connection
+
+[[!DNL Mailchimp]](https://mailchimp.com) *(also known as [!DNL Intuit Mailchimp])* is a popular marketing automation platform and email marketing service used by businesses to manage and talk to contacts *(clients, customers, or other interested parties)* using mailing lists and email marketing campaigns.
+
+[!DNL Mailchimp Interest Categories] uses [audiences](https://mailchimp.com/help/getting-started-audience/), [groups](https://mailchimp.com/help/getting-started-with-groups/) and interest categories *(also known as group names or group titles)*. Each [!DNL Mailchimp] group is a list of interest categories. Contacts are associated with an interest category when they subscribe to one or more interest category through a signup form on your website. Within an audience you can also organize the contacts into groups and associate them with interest categories, these can then be used to make segments. You can use these segments to broadcast targeted campaign emails to the subscribed contacts.
```

Comment:
Great context, I very much appreciate this section! Just one quick comment about a comma and minor wording revision:

Within an audience, you can also organize the contacts into groups and associate them with interest categories, and these can then be used to create segments. You can use these segments to broadcast targeted campaign emails to the subscribed contacts.